data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/beaconhandlers/GenesisTimeHandler.java

```diff
@@ -35,7 +33,12 @@ public String getPath() {
   }
 
   @Override
-  public String handleRequest(RequestParams params) {
-    return returnString;
+  public Object handleRequest(RequestParams params) {
+    Map<String, Object> jsonObject = new HashMap<>();
```

Comment:
Looks like this should just return an `Integer`: https://github.com/ethereum/eth2.0-APIs/blob/48095af5237ae0c0d58bd24a32b541065b31a9b0/apis/validator/beacon-node-oapi.yaml#L296-L300 ?

Reply:
Even easier... Done.